.eslintrc

```diff
@@ -10,7 +14,12 @@
     "newline-after-var": [0],
     "react/jsx-indent-props": [2, 2], // no tabs, indent is two spaces
     "react/jsx-key": [2], // validate that key prop exists
-    "react/jsx-no-undef": [2] // disallow undeclared variables in JSX
+    "react/jsx-no-undef": [2], // disallow undeclared variables in JSX
+    "no-unused-vars": [ 1, {
+      "vars": "local",
+      "argsIgnorePattern": "action"
+    }],
+    "comma-dangle": [0]
```

Comment:
It's a common pattern in Python, encouraged because lists like this

    [
      'a',
      'b',
      'c',
    ]

Can be updated with one line diffs more easily. I'd vote to silence the warning